ui/src/views/compute/backup/BackupSchedule.vue

```diff
@@ -147,18 +147,15 @@ export default {
   mounted () {
     this.dataSchedules = []
     if (this.dataSource && Object.keys(this.dataSource).length > 0) {
-      this.dataSchedules.push(this.dataSource)
+      this.dataSchedules = this.dataSource
```

Comment:
Yes, that's right. Earlier, i.e before this PR, backup schedules was always 1 per instance due to a limitation (https://github.com/apache/cloudstack/issues/9344) . But now that it is a list, this would cause issues.